api/clients/v2/coretypes/payload_to_blob_test.go

```diff
@@ -0,0 +1,62 @@
+// nolint: lll // Example contains long lines to print output
+package coretypes
+
+import (
+	"encoding/hex"
+	"fmt"
+
+	"github.com/Layr-Labs/eigenda/encoding"
+)
+
+// Example demonstrating the conversion process from a payload, to an encodedPayload interpreted as
+// evaluations of a polynomial, which is then IFFT'd to produce a Blob in coefficient form.
+// This example demonstrates the process that [Payload.ToBlob] performs internally.
+func Example_payloadToBlobConversion() {
+	// We create a payload of 2 symbols (64 bytes), which with a payload head of 1 symbol (32 bytes),
```

Comment:
nit: I'm assuming "payload head" is a typo. Elsewhere I believe we've used `encoded payload header`?

Reply:
Oh did we? Nice I'll update. I also found PayloadHeader sounded wrong, but I thought I remembered we were calling it PayloadHeader for some reason.

Reply:
[1400d66](https://github.com/Layr-Labs/eigenda/pull/1833/commits/1400d661e2e2a10031c14b0699d9727f69d2698e)